plugins/vpp/ifplugin/vppcalls/dump_nat_vppcalls.go

```diff
@@ -29,22 +29,23 @@ import (
 func (handler *natVppHandler) Nat44GlobalConfigDump(swIfIndices ifaceidx.SwIfIndex) (*nat.Nat44Global, error) {
 	handler.log.Debug("dumping Nat44Global")
 	// Dump all necessary data to reconstruct global NAT configuration
-	isEnabled, err := handler.nat44IsForwardingEnabled()
+	isEnabled, err := handler.isForwardingEnabled()
 	if err != nil {
 		return nil, err
 	}
 	natInterfaces, err := handler.Nat44InterfaceDump(swIfIndices)
 	if err != nil {
 		return nil, err
 	}
-	natOutputFeature, err := handler.nat44InterfaceOutputFeatureDump(swIfIndices)
+	natOutputFeature, err := handler.interfaceOutputFeatureDump(swIfIndices)
 	if err != nil {
 		return nil, err
 	}
-	natAddressPools, err := handler.nat44AddressDump()
+	natAddressPools, err := handler.addressDump()
```

Comment:
There is also dump of pool addresses for NAT64.. how are we going to know difference?

Reply:
This applies to other cases as well.. why getting rid of the nat44? There are clearly settings for NAT64 and NAT44.. it might be confusing to reader to not know which one is being called.